tests/UpgradeRule/PHP/fixtures/class-to-traits.testfixture

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can we add some more complex example e.g.: an abstract class with some implement clauses and some methods.